app/music_list.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './helper'
+require_relative './music_album'
+
+module MusicList
+  include Helper
+  def list_music_albums
+    if @music_albums.empty?
+      puts 'You don\'t have any Music Albums.'
+    else
+      @music_albums.each_with_index do |music_album, index|
+        # rubocop:disable Layout/LineLength
+        puts "#{index}) Album: #{music_album.name}, Genre: #{@genres.empty? ? 'No genre' : music_album.genre.name}, Publish Date: #{time_format(music_album.publish_date)}, Archived: #{music_album.archived}, Spotify: #{music_album.on_spotify}"
+        # rubocop:enable Layout/LineLength
+      end
+    end
+  end
+
+  def add_music_album
+    print 'Album Name: '
+    name = gets.chomp
+    year = select_year
+    month = select_month
+    day = select_day
+
```

Comment:
My dear friend, you have forgotten the `on_spotify` attribute from MusicAlbums class.